style.css

```diff
@@ -0,0 +1,795 @@
+    grid-gap: 15px;
+    grid-template-columns: 1fr;
+  }
+  .recommended-video-container-U2B {
+    grid-column: 1;
+    grid-row: 2;
+
+    display: grid;
+    grid-gap: 15px;
+    grid-template-rows: 50px 150px 150px 150px 150px 150px 150px;
+    grid-template-columns: 1fr;
+  }
+
+  .video-section {
+    grid-column: 1;
+    grid-row: 1;
+  }
+  .video-title {
+    grid-column: 1;
+    grid-row: 2;
+  }
+  .video-description {
+    grid-column: 1;
+    grid-row: 3;
+  }
+  .video-comments {
+    grid-column: 1;
+    grid-row: 4;
+  }
+  .recommended-video {
+    grid-column: 1;
+    grid-row: auto;
+  }
+}
+
+.background-color-white {
+  background-color: #fff;
+}
+
+/**/
+.center {
+  margin: auto;
+  padding: 10px;
+}
+
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 40px;
+  height: 15px;
+}
+
+/* Hide default HTML checkbox */
+.switch input {
+  opacity: 0;
+  width: 0;
+  height: 0;
+}
+
+/* The slider */
+.slider {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #ccc;
+  -webkit-transition: 0.4s;
+  -moz-transition: 0.4s;
+  -o-transition: 0.4s;
+  transition: 0.4s;
+}
+
+.slider:before {
+  position: absolute;
+  content: "";
+  height: 15px;
+  width: 15px;
+  left: 2px;
+  bottom: 0px;
+  background-color: white;
+  -webkit-transition: 0.4s;
+  -moz-transition: 0.4s;
+  -o-transition: 0.4s;
+  transition: 0.4s;
+}
+
+input:checked + .slider {
+  background-color: #2196f3;
+}
+
+input:focus + .slider {
+  box-shadow: 0 0 1px #2196f3;
+}
+
+input:checked + .slider:before {
+  -webkit-transform: translateX(26px);
+  -ms-transform: translateX(26px);
+  -moz-transform: translateX(26px);
+  -o-transform: translateX(26px);
+  transform: translateX(26px);
+}
+
+/* Rounded sliders */
+.slider-round {
+  border-radius: 34px;
+}
+
+.slider-round:before {
+  border-radius: 50%;
+}
+
+.miniature-videos {
+  width: 230px;
+  height: 120px;
+  position: relative;
+  margin: 0 auto;
+}
+
+.duration-miniature {
+  position: absolute;
+  top: 80%;
+  left: 82%;
+  background-color: black;
+  color: white;
+  z-index: 2;
+  opacity: 0.8;
+}
+
+.top-margin-white {
+  background-color: white;
+  margin-top: 10px;
+}
+
+.icon-loading {
+  text-align: center;
+}
+
+.nav-styles {
+  background-color: white;
+  padding: 10px;
+}
+
+.youtube-icon-size {
+  width: 100px;
+  height: 30px;
+  padding: 10px;
+}
+
+.user-image-size {
+  width: 32px;
+  height: 32px;
+}
+
+.publish-styles {
+  margin-top: 10px;
+  font-weight: 700;
+}
+
+.hr-styles {
+  margin: 0;
+  border-color: lightgrey;
+}
+
+.show-more-styles {
+  text-align: center;
+  color: rgb(96, 96, 96);
+  font-weight: 700;
+}
+
+.loading-styles {
+  width: 20px;
+  height: 20px;
+}
+
+.aside-styles {
+  background-color: white;
+  padding: 5px;
+}
+
+@media (max-width: 991px) {
+  .aside-styles {
+    background-color: white;
+    padding: 5px;
+    margin-top: 15px;
+  }
+
+  .no-border:active {
+  }
```

Comment:
Line number 343 Do not use empty rulesets, better to delete

Reply:
Done!. Thank you Anna.